app/cdap/components/RecipeDetails/DrawerHeader/index.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import { grey } from '@material-ui/core/colors';
+import React from 'react';
+import styled from 'styled-components';
+import CloseIcon from '@material-ui/icons/Close';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import EditIcon from '@material-ui/icons/Edit';
+import SaveAltIcon from '@material-ui/icons/SaveAlt';
+import { Box, IconButton, Typography } from '@material-ui/core';
+import T from 'i18n-react';
+import { UnderlineIcon } from 'components/RecipeDetails/IconStore/UnderlineSVG';
+
+interface IDrawerHeaderProps {
+  onCloseDetail: () => void;
+}
+
+const MainHeadWrapper = styled(Box)`
+  margin-top: 20px;
+`;
+
+const HeadWrapper = styled(Box)`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const RecipeHeaderLabel = styled(Typography)`
+  font-weight: 400;
+  font-size: 20px;
+  margin-bottom: 6px;
+  color: ${grey[900]};
+  padding: 1px;
+`;
+
+const IconsWrapper = styled(Box)`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledCommonIconButton = styled(IconButton)`
+  padding: 12px;
+`;
+
+const StyledEditIcon = styled(EditIcon)`
```

Comment:
use multiple inheritance
check below link 
https://github.com/styled-components/styled-components/issues/773#:~:text=I%27d%20suggest%20you%20to%20use%20mixins%20with%20the%20css%20helper%20in%20this%20case%2C%20since%20it%27s%20not%20really%20an%20inheritance%20problem%2C%20but%20a%20compositional%20problem.

we need to confirm this with radhika and team, so once we implement this we need to keep an eye whether they accept is or not

do the following
defined styles using CSS helper from styled-components
then use this styles for both icons as shown in the link above

Reply:
Done